config-templates/google/gcp-compute-persistent-disk-csi-driver/1.8.0/metadata.json

```diff
@@ -0,0 +1,73 @@
+{
+    "name": "1.8.0",
+    "revision": "1",
+    "enabled": "true",
+    "status": "supported",
+    "description": "GCP Compute Persistent Disk CSI Driver",
+    "tags": [
+        {
+            "name": "provider",
+            "value": "google"
+        },
+        {
+            "name": "offering-name",
+            "value": "gcp-compute-persistent-disk-csi-driver"
+        },
+        {
+            "name": "offering-type",
+            "value": "non-sds"
+        },
+        {
+            "name": "provider-url",
+            "value": "https://cloud.google.com/gcp"
+        },
+        {
+            "name": "support-msg",
+            "value": "For support contact Google support"
+        },
+        {
+            "name": "provider-logo",
+            "value": "logo/gcp-compute-persistent-disk-csi-driver.svg"
+        },
+        {
+            "name": "supported-envs",
+            "value": "Google"
+        },
+        {
+            "name": "supported-k8s-versions",
+            "value": "1.22, 1.23, 1.24, 1.25"
+        },
+        {
+            "name": "supported-ocp-versions",
+            "value": "4.9, 4.10, 4.11"
```

Comment:
is this template  supported on 4.7 & 4.8 as well @gauravAggrahari ?
```
OpenShift Versions   
4.7.59_openshift (deprecated, unsupported in 37 days)   
4.8.51_openshift   
4.9.49_openshift   
4.10.36_openshift (default)   
4.11.8_openshift   
```

Reply:
@anandkarra , 
We can remove this check for now.

Reply:
Removed here: https://github.com/IBM/ibm-satellite-storage/pull/221/commits/0136711b3cf6db58933fa058758dbd71e248510b